CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ so we can correct that!
 See [`HACKING_QUICKSTART.md`](docs/HACKING_QUICKSTART.md) for more information
 on how to start working on Servo.
 
-## Pull Request Checklist
+## Pull request checklist
```

Comment:
I think this is a difference between North America and Europe title-casing: https://www.uoc.edu/portal/en/servei-linguistic/convencions/british-american-english/capitalization/index.html

I guess we should standardize on one of these styles.

Reply:
haha, i think we’ve run into this before. regional conventions aside, i think title case should be avoided because it destroys the details and nuance conveyed by capitalisation.